src/Sylius/Bundle/MoneyBundle/Twig/ConvertMoneyExtension.php

```diff
@@ -13,20 +13,20 @@
 
 namespace Sylius\Bundle\MoneyBundle\Twig;
 
-use Sylius\Bundle\MoneyBundle\Templating\Helper\ConvertMoneyHelperInterface;
+use Sylius\Component\Currency\Converter\CurrencyConverterInterface;
 use Twig\Extension\AbstractExtension;
 use Twig\TwigFilter;
 
 final class ConvertMoneyExtension extends AbstractExtension
 {
-    public function __construct(private ConvertMoneyHelperInterface $helper)
+    public function __construct(private CurrencyConverterInterface $currencyConverter)
```

Comment:
I think you shoud add a note on the UPGRADE guide for that too.

Reply:
And for `src/Sylius/Bundle/MoneyBundle/Twig/FormatMoneyExtension.php` as well